201-cosmosdb-advanced-threat-protection-create-account/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "type": "QuickStart",
+    "itemDisplayName": "Create a CosmosDB Account with Advanced Threat Protection.",
```

Comment:
(preview)

Reply:
itemDisplayName is limited to 60 chars, and the current text is 58 chars.